src/hypothesis/extra/pandas/impl.py

```diff
@@ -156,8 +157,8 @@ def range_indexes(min_size=0, max_size=None):
       it will default to some suitable value based on min_size.
 
     """
-    st.check_valid_size(min_size, 'min_size')
-    st.check_valid_size(max_size, 'max_size')
+    check_valid_size(min_size, 'min_size')
+    check_valid_size(max_size, 'max_size')
```

Comment:
Why have you changed some of these calls, and not others? Eg L164. If we’re doing it, let’s do it all at once!

Reply:
I changed this one because `check_valid_size` was no longer in the strategy namespace and thus was erroring. I've now changed them all though.